packages/dd-trace/src/telemetry/send-data.js

```diff
@@ -54,8 +56,19 @@ function sendData (config, application, host, reqType, payload = {}) {
     host
   })
 
-  request(data, options, () => {
-    // ignore errors
+  request(data, options, (error) => {
+    if (error && process.env.DD_API_KEY) {
+      const backendHeader = { 'DD-API-KEY': process.env.DD_API_KEY, ...options.headers }
+      const backendOptions = {
+        url: 'https://all-http-intake.logs.datad0g.com/api/v2/apmtelemetry',
+        headers: backendHeader,
+        ...options
+      }
```

Comment:
Doing the spread last would overwrite the custom field if present in the original, so the url here would not use the backed url.